src/ui/components/filters/filteroptionscomponent.js

```diff
@@ -313,6 +386,28 @@ export default class FilterOptionsComponent extends Component {
     });
   }
 
+  getLocationRadiusFilterNode () {
+    const selectedOption = this.config.options.find(o => o.selected);
+    if (!selectedOption) {
+      return FilterNodeFactory.from();
```

Comment:
This is a minor nit, but do we want to return you anything if no option was selected?

Reply:
I thought maybe it was simpler to just return an empty filternode? alternatively I could add a clearLocationRadiusFilterNode() method